backend/src/core/models/games.aggregate.js

```diff
@@ -8,40 +8,40 @@ export class GamesAggregate {
   }
 
   get content() {
-    return this.#games;
+    const contentCopy = new GamesAggregate(this.#games).#games;
```

Comment:
Not a correct copy => `Game.fromDbEntry(game)` does not properly copy the props for `developers` and `genres`. It will pass them via reference which is dangerous. Use `deepCopy`.